src/stactools/viirs/stac.py

```diff
@@ -136,6 +136,11 @@ def create_collection(product: str) -> Collection:
 
     ScientificExtension.add_to(collection)
     collection.extra_fields["sci:doi"] = collection_fragments["sci:doi"]
+    collection.extra_fields["sci:citation"] = collection_fragments["sci:citation"]
+    if collection_fragments.get("sci:publications", None):
```

Comment:
```suggestion
    if "sci:publications" in collection_fragments:
```

I _think_ this is more idiomatic.

Reply:
Yep. I like. Done.